src/main/kotlin/cacophony/pipeline/CacophonyPipeline.kt

```diff
@@ -235,4 +237,40 @@ class CacophonyPipeline(
         asm.forEach { (function, asm) -> println("$function generates asm:\n$asm") }
         return asm
     }
+
+    fun generateAsm(input: Input, dest: Path) {
```

Comment:
This should not be an overload of regular `generateAsm(input)`, their goals are quite different.

Reply:
Can you be more specific?

Reply:
One of them generates the asm, the second one generates asm and writes to some file. Maybe saveAsm would be a better name?

Reply:
I just separated them entirely as `saveAsm` shouldn't be generating anything while something like `generateAndSaveAsm` is verbose